docs/configuration/outbox-event-router.asciidoc

```diff
@@ -0,0 +1,195 @@
+====
+
+[NOTE]
+====
+This SMT does *not* support the MongoDB connector.
+====
+
+The Outbox Event Router SMT has the intent to provide out-of-the-box support for the Outbox Pattern.
+For more information on why and how the Outbox Pattern is used please refer our blog post link:/blog/2019-02-19-reliable-microservices-data-exchange-with-the-outbox-pattern/[Reliable Microservices Data Exchange With the Outbox Pattern]
+
+For more implementation details please refer to the https://issues.jboss.org/browse/DBZ-1169[Outbox SMT Design]
+
+A working example can also be found at our https://github.com/debezium/debezium-examples[Examples Repository] in the https://github.com/debezium/debezium-examples/tree/master/outbox[`outbox` directory].
+
+=== Example
+
+In order to understand the configuration and terms used in this SMT, let's look into its parts with the given expected outbox message:
+
+[source,javascript,indent=0]
+----
+# Kafka Topic: outbox.event.order
+# Kafka Message key: "1"
+# Kafka Message Headers: "id=4d47e190-0402-4048-bc2c-89dd54343cdc"
+# Kafka Message Timestamp: 1556890294484
+{
+  "eventType": "OrderCreated",
+  "payload": "{\"id\": 1, \"lineItems\": [{\"id\": 1, \"item\": \"Debezium in Action\", \"status\": \"ENTERED\", \"quantity\": 2, \"totalPrice\": 39.98}, {\"id\": 2, \"item\": \"Debezium for Dummies\", \"status\": \"ENTERED\", \"quantity\": 1, \"totalPrice\": 29.99}], \"orderDate\": \"2019-01-31T12:13:01\", \"customerId\": 123}"
+}
+----
+
+This message is generated by transforming a Debezium raw message, which looks like:
+
+[source,javascript,indent=0]
+----
+# Kafka Message key: "406c07f3-26f0-4eea-a50c-109940064b8f"
+# Kafka Message Headers: ""
+# Kafka Message Timestamp: 1556890294484
+{
+  "before": null,
+  "after": {
+    "id": "406c07f3-26f0-4eea-a50c-109940064b8f",
+    "aggregateid": "1",
+    "aggregatetype": "Order",
+    "payload": "{\"id\": 1, \"lineItems\": [{\"id\": 1, \"item\": \"Debezium in Action\", \"status\": \"ENTERED\", \"quantity\": 2, \"totalPrice\": 39.98}, {\"id\": 2, \"item\": \"Debezium for Dummies\", \"status\": \"ENTERED\", \"quantity\": 1, \"totalPrice\": 29.99}], \"orderDate\": \"2019-01-31T12:13:01\", \"customerId\": 123}",
+    "timestamp": 1556890294344,
+    "type": "OrderCreated"
+  },
+  "source": {
+    "version": "0.9.3.Final",
+    "connector": "postgresql",
+    "name": "dbserver1-bare",
+    "db": "orderdb",
+    "ts_usec": 1556890294448870,
+    "txId": 584,
+    "lsn": 24064704,
+    "schema": "inventory",
+    "table": "outboxevent",
+    "snapshot": false,
+    "last_snapshot_record": null,
+    "xmin": null
+  },
+  "op": "c",
+  "ts_ms": 1556890294484
+}
+----
+
+This result was achieved with the link:#default-table-columns[Default Configuration] which assumes a table structure and event routing based on aggregates, in case you want a custom behavior, the smt is fully configurable, check the link:#configuration-options[Available configuration options].
```

Comment:
```suggestion
This result was achieved with the link:#default-table-columns[default configuration] which assumes a table structure and event routing based on aggregates. In case you want a custom behavior, the SMT is fully configurable, check the link:#configuration-options[available configuration options].
```